Source/WebKit/WebProcess/DigitalCredentials/DigitalCredentialsCoordinator.cpp

```diff
@@ -51,27 +49,20 @@ RefPtr<WebPage> DigitalCredentialsCoordinator::protectedPage() const
     return m_page.get();
 }
 
-void DigitalCredentialsCoordinator::requestDigitalCredential(const LocalFrame& frame, const DigitalCredentialRequestOptions& options, DigitalCredentialRequestCompletionHandler&& handler)
+void DigitalCredentialsCoordinator::showDigitalCredentialsPicker(const WebCore::DigitalCredentialsRequestData& request, WTF::CompletionHandler<void(Expected<WebCore::DigitalCredentialsResponseData, WebCore::ExceptionData>&&)>&& completionHandler)
 {
-    RefPtr webFrame = WebFrame::fromCoreFrame(frame);
-    RefPtr page = m_page.get();
-    if (!webFrame || !page) {
-        LOG_ERROR("Unable to get frame or page");
-        handler(ExceptionData { ExceptionCode::InvalidStateError, "Unable to get frame or page"_s });
-        return;
-    }
-    page->sendWithAsyncReply(Messages::DigitalCredentialsCoordinatorProxy::RequestDigitalCredential(webFrame->frameID(), webFrame->info(), options), WTFMove(handler));
+    if (auto page = protectedPage())
```

Comment:
```suggested
    if (RefPtr page = protectedPage())
```
do not use `auto` if the type is Ref, RefPtr or RetainPtr (new code hardening style)